python/test/t_IntegrationExpansion_std.py

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import math as m
+
+ot.TESTPREAMBLE()
+
+
+# Problem parameters
+dimension = 3
+a = 7.0
+b = 0.1
+# Reference analytical values
+meanTh = a / 2
+covTh = (b**2 * m.pi**8) / 18.0 + (b * m.pi**4) / 5.0 + (a**2) / 8.0 + 1.0 / 2.0
+sob_1 = [
+    (b * m.pi**4 / 5.0 + b**2 * m.pi**8 / 50.0 + 1.0 / 2.0) / covTh,
+    (a**2 / 8.0) / covTh,
+    0.0,
+]
+sob_2 = [0.0, (b**2 * m.pi**8 / 18.0 - b**2 * m.pi**8 / 50.0) / covTh, 0.0]
+sob_3 = [0.0]
+sob_T1 = [
+    sob_1[0] + sob_2[0] + sob_2[1] + sob_3[0],
+    sob_1[1] + sob_2[0] + sob_2[2] + sob_3[0],
+    sob_1[2] + sob_2[1] + sob_2[2] + sob_3[0],
+]
+sob_T2 = [
+    sob_2[0] + sob_2[1] + sob_3[0],
+    sob_2[0] + sob_2[2] + sob_3[0],
+    sob_2[1] + sob_2[2] + sob_3[0],
+]
+sob_T3 = [sob_3[0]]
+# Create the Ishigami function
+inputVariables = ["xi1", "xi2", "xi3"]
```

Comment:
preferable to use `openturns.usecases` to shorten the test

Reply:
Done